tests/mock_network/utils.rs

```diff
@@ -94,34 +90,15 @@ impl TestNode {
         unwrap!(self.inner.our_prefix(), "{}", self.name())
     }
 
-    pub fn in_src_location(&self, src: &SrcLocation) -> bool {
-        self.inner.in_src_location(src)
-    }
-
-    pub fn in_dst_location(&self, dst: &DstLocation) -> bool {
-        self.inner.in_dst_location(dst)
-    }
-
-    pub fn env(&self) -> &Environment {
-        &self.env
-    }
-
     pub fn poll(&mut self) -> bool {
         let mut result = false;
 
-        // Exhaust all the events/actions from the channels but return true only if at least one of
-        // those events/actions are considered as handled (that is there is at least one
-        // non-timeout).
```

Comment:
why the comment removed?
any change of the situation?

Reply:
Yes, I removed the hack where we ignored timeout events so the comment is no longer necessary.